repos/spack_repo/builtin/packages/chapel/package.py

```diff
@@ -196,6 +197,14 @@ class Chapel(AutotoolsPackage, CudaPackage, ROCmPackage):
         values=("gasnet", "none", "ofi", "ugni"),
     )
 
+    variant(
+        "comm_ofi_oob",
+        values=("sockets", "mpi", "pmi2", "unset"),
+        default="unset",
+        description="Select out-of-band support",
+        multi=False,
```

Comment:
This variant should be conditional on `comm=ofi`
```suggestion
        multi=False,
        when="comm=ofi",
```